server/rematch/settings.py

```diff
@@ -23,19 +24,24 @@
 SECRET_KEY = 'qvc7t@rd5#1l-n_%%&+_fu+-lu#sp2oonf9mto%bn-1#i7$(tu'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+# FIXME
+if "celeryd" in sys.argv or "celery" in sys.argv:
+  DEBUG = False
+else:
+  DEBUG = True
 
 
 # As of django 1.10, allowed hosts are validated in debug as well,
 # this disables that and makes sure all hosts are acceptible when
 # running in debug mode. for more details see
 # https://docs.djangoproject.com/en/1.10/ref/settings/
 # for security implications see
-# https://docs.djangoproject.com/en/1.10/topics/security/#host-headers-virtual-hosting
+# https://docs.djangoproject.com/en/1.10/topics/security/ \
+# #host-headers-virtual-hosting
 if DEBUG:
   ALLOWED_HOSTS = ['*']
 else:
-  ALLOWED_HOSTS = []
+  ALLOWED_HOSTS = ['127.0.0.1', '172.18.0.*', '192*']
```

Comment:
These are the ranges exposed to let external users (aka machines which are not inside Docker) to access
the Docker network.